man/systemd-analyze.xml

```diff
@@ -744,6 +744,15 @@ Service b@0.service not loaded, b.socket cannot be started.
         generators enabled will generally result in some warnings.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--ignore-dependencies</option></term>
+
+        <listitem><para>Ignore unit dependencies. When the manager object loads a unit, it recursively loads
```

Comment:
I'd slightly reword this - specify that this is used by the `verify` verb, and also don't mention the manager object - that's an internal implementation detail useful to us developers, but that we don't mention in user-facing documentation. Just mention something along the lines of `When systemd-analyze verify loads..."